src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java

```diff
@@ -34,10 +36,26 @@ public void adsAppNexusVideoAdIsDisplayedIfAbcdAdIsTargetedOnThisSameArticleOasi
       groups = {"AdsAbcdAdIsDisplayedIfBidersAreDisabledOasis", "AbcdProductPriorityOasis"}
   )
   public void adsAbcdAdIsDisplayedIfBidersAreDisabledOasis() {
-    AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(WIKIA, URL_ARTICLE));
+    AdsBaseObject ads = new AdsBaseObject(driver, TEST_PAGE.getUrl());
     verifyAbcdAdIsDisplayed(ads);
   }
 
+  @Test(
+      groups = {"AdsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage", "AbcdProductPriorityOasis"}
+  )
+  public void adsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage() {
+    AdsBaseObject ads = new AdsBaseObject(driver, FV_PAGE.getUrl());
+    verifyAbcdAdIsNotDisplayed(ads);
+
+    Assert.assertTrue(ads.slotHasSize(AdsContent.TOP_LB, 728, 90));
+    verifyNoUAPSizesInSlot(ads, AdsContent.TOP_LB);
+  }
+
+  private void verifyNoUAPSizesInSlot(AdsBaseObject ads, String slot) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyNoUAPSizesInSlot" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)